### PR TITLE
Add and test colors and fonts from Tailwind

### PR DESCRIPTION
**Pull request by liubomyr123**

### CODE REVIEW

File: src/app/layouts/global/Header.tsx [hunk 1]
```diff
@@ -84,12 +84,17 @@ function LanguageToggle (): JSX.Element {
...
       {(["en", "uk"] as TranslationKeys[]).map((language, index) => (
         <button
+          className={index === 0 ? "text-primary-200" : "text-primary-500"}
           key={language}
           name={language}
           onClick={handleChangeLanguage}
         >
           {t(language)}
-          {index === 0 ? "|" : ""}
+          {index === 0
```
Comment (chekl) [inline src/app/layouts/global/Header.tsx:93]: delete changes from the header

File: src/app/layouts/global/Footer.tsx [hunk 2]
```diff
@@ -4,7 +4,7 @@ export default function Footer (): JSX.Element {
   const t = useTypedTranslation();
 
   return (
-    <footer className="p-3 bg-[#D9D9D9]">
+    <footer className="p-3 bg-background-footer">
```
Comment (chekl) [inline src/app/layouts/global/Footer.tsx:7]: delete changes from the footer

**Review by chekl — COMMENTED** [review]
delete changes from the header and footer

**Comment by liubomyr123** [comment]
@chekl Check please my changes - I revert styles for Footer and Header

**Pull request merged**